### DISCUSSION TITLE
Cheats from a custom database don't show up

### DISCUSSION THREAD

**Discussion by ANN-4-BE-11-A** [rafaelvcaetano/melonDS-android#948] [Q&A]
I'm trying to use this custom database for a Pokemon romhack: https://drive.google.com/file/d/1NyGXX35sBpCECL2YHDDRuJOO2ab_1NtO/view?usp=drivesdk However, when I boot up the game and go into `Cheats`, I'm told that there are no cheats available. What am I doing wrong?

**Comment by rafaelvcaetano** [comment]
Hey! I'm the one you spoke with in the forum. I see that you also attempted my initial suggested. Could you tell me the name of the romhack so that I can test on my side?

**Reply by ANN-4-BE-11-A** [reply]
Pokemon Sacred Gold. You can find the official page here: https://gbatemp.net/threads/pokemon-sacred-gold-storm-silver.327567/

**Comment by rafaelvcaetano** [comment]
So, it looks like there is a problem with the cheat parser. It will only load cheats that are inside folders. Cheats that are placed inside the root of the game won't be loaded. Here is a version of the database you provided that will work with the current melonDS version: https://pastebin.com/XS9ZhBzd

**Reply by ANN-4-BE-11-A** [reply]
Neat. Though I decided not to play the romhack, so I won't test this. Maybe this can help others, however.
